### PR TITLE
Privacy.md

### PR DESCRIPTION
**Pull request by HarryHeights**
This is the privacy policy I proposed yesterday. Something's might be off but I'm not entirely sure you'll find any mistakes. Please review and see if there are any modifications to make to the document. Thanks.

### CODE REVIEW

File: Privacy.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+## Types of Data Collected
+
+### Personal Data
+
+Due to the integration of OpenWeatherMap API in the software personal data may be collected while the weather widget is in use. This data is in no way retrieved by the developer of Weather widget.
+
+### Usage Data
+
+This app does not record usage information, however, usage statistics are captured by OpenWeatherMap with respect to their [policy](https://openweather.co.uk/privacy-policy).
```
Comment (sanderbaas) [inline Privacy.md]: Can you maybe change this to the .org url? https://openweathermap.org/privacy-policy

File: Privacy.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+With the integration of OpenWeatherMap API, the data collected is used to allow Weather widget 
+
+- Access current weather data for any location including over 200,000 cities.
+
+---
+
+## Security Of Data
+
+There are no security risks that may likely occur while using Weather Widget software. However, with the inclusion of OpenWeatherMap API, it is necessary to read how they(OpenWeatherMap) secure your data that may be collected while you utilize Weather Widget. The developers of Phone Saver assume no responsibility for data integrity issues caused by using this application.
```
Comment (sanderbaas) [inline Privacy.md]: Can you change Phone Saver to Weather Widget?
And maybe add a space after they and before the '(OpenWeatherMap)'?

File: Privacy.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+---
+
+## Changes To This Privacy Policy
+
+This privacy policy statement is not final and is subject to changes at later times. This is done to accommodate changes in the software's architecture due to further developments and that these changes may not align with several statements and policies stated in the document. With that stated, you are advised to check this policy statement occasionally for any changes that may have occurred. Changes to this policy become effective as soon as they are made. Also occasionally check policies of third-party software that is linked with Weather widget.
```
Comment (sanderbaas) [inline Privacy.md]: Can you uppercase the 'w' in the last word, to keep in line with the other mentions?

File: Privacy.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+
+---
+
+
+## Disclaimer   
+
+Weather widget is developed in good faith to help end users keep tabs of weather variations at different intervals. The software has an external dependency(OpenWeatherMap API) that might collect some information when you utilize Weather widget. This is not directly controlled by the developer of Weather widget neither does the developer has oversight of the operations of OpenWeatherMap hence, the developer assumes no responsibility for any losses and/or damages of data that are in connection with the use of this software.
```
Comment (sanderbaas) [inline Privacy.md]: Could you add a space after dependency?

**Comment by HarryHeights** [comment]
Changes have been made.

**Pull request merged**